### ISSUE TITLE
The link to the person form on conduct test card is inaccessible

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4066]
## Area or Component
Conduct tests - test card

## Quick Summary
The person's name on the card is a link to their details form. This link is not keyboard accessible (you can't tab to it) making it unusable for screenreader users.

### How to Replicate
Tab through the conduct test page and see there's no way to hit the person name link.

## Accessibility Violation
Action cannot be performed by keyboard alone: 
https://cdc-devtools.dequecloud.com/rule-help/en/keyboard-inaccessible

## Source code
https://github.com/CDCgov/prime-simplereport/blob/f11273c49f8e25f8a85449b510e6947a49d3457c/frontend/src/app/testQueue/QueueItem.tsx#L783-L793

## Action Requested
Change the link into either a proper `<a>` or `<button>` so that it receives keyboard focus.

## Original Deque Issue
https://docs.google.com/spreadsheets/d/1b6xjubAN5Uw11DV4_B5zvSu7HeDI6Z-DpV4DMcToqAY/edit#gid=981460209&range=8:8
